main.nf

```diff
@@ -613,7 +613,7 @@ if(params.aligner == 'hisat2'){
         file "where_are_my_files.txt"
 
         script:
-        index_base = hs2_indices[0].toString() - ~/.\d.ht2/
+        index_base = hs2_indices[0].toString() - ~/(.\d.ht2)?(.\d.ht2l)?/
```

Comment:
Can we not just have this?
```suggestion
        index_base = hs2_indices[0].toString() - ~/.\d.ht2l?/
```

Reply:
Yeah but what happens with indices having just the name `.ht2` ?

Reply:
The question mark makes the preceding character optional:
<img src="https://user-images.githubusercontent.com/465550/52786623-a599fb80-305b-11e9-9996-ff9425ee16ad.png" height=300>

(screengrab from https://regex101.com/ )

Reply:
I need to bookmark and use that site more, this looks nice :-)

Reply:
I use it a _lot_ 😉